geoips/interfaces/base.py

```diff
@@ -331,9 +330,18 @@ def __new__(cls):
 
     def __init__(self):
         """YAML plugin interface init method."""
-        self._unvalidated_plugins = self._create_unvalidated_plugins_cache(
-            load_all_yaml_plugins()
-        )
+        try:
+            self._unvalidated_plugins = load_all_yaml_plugins()
+        except PluginRegistryError:
+            self._unvalidated_plugins = {}
```

Comment:
@mindyls @evrose54 Is this the correct behavior? If there is a PluginRegistryError, just set this to `{}` and continue? Why not allow the error to be raised?

Reply:
This is an @mindyls as she created it. Personally I would continue to raise the error.

Reply:
After talking with @mindyls there is a reason for this. This will be fixed when we implement a caching `PluginRegistry` class to replace `load_all_yaml_plugins`. I'll write an issue for that and point to this section of code as something that should be updated.

Reply:
If you raise the error, then you can never create the plugin registries.  Since create_plugin_registry imports from geoips, and this gets called on import of anything from geoips, the error gets raised BEFORE creating the plugin registries.  This needs to be addressed in the issue @jsolbrig is going to write about creating a separate plugin registry class for caching the plugins (which would also ensure the plugins registry is not loaded immediately on import).

Reply:
I created #389 to address this.